ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/EngineNewPayloadV2.java

```diff
@@ -64,7 +64,7 @@ protected ValidationResult<RpcErrorType> validateParameters(
     }
     if (payloadParameter.getExcessBlobGas() != null) {
       return ValidationResult.invalid(
-          RpcErrorType.INVALID_PARAMS, "non-null ExcessBlobGas pre-cancun");
+          RpcErrorType.INVALID_EXCESS_BLOB_GAS_PARAMS, "Missing excess blob gas field");
```

Comment:
do we want to keep the "pre-cancun" text

Reply:
Can do if you think it's beneficial? These messages are only seen in the logs though.